vnext/Desktop/React.Windows.Desktop.vcxproj.filters

```diff
@@ -84,9 +84,6 @@
     <Midl Include="ABI\TestController.idl">
       <Filter>ABI</Filter>
     </Midl>
-    <Midl Include="..\Microsoft.ReactNative\ReactInstance.idl">
```

Comment:
caused by roundtripping in VS.

Reply:
Could you undo it? VS often does strange thing to out filters files. I often must restart VS just because of it.

---
In reply to: [455502717](https://github.com/microsoft/react-native-windows/pull/5533#discussion_r455502717) [](ancestors = 455502717)